platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PreConsensusEventFile.java

```diff
@@ -191,13 +297,28 @@ public PreConsensusEventMutableFile getMutableFile() throws IOException {
      *
```

Comment:
Perhaps add a note to the javadoc that this method permanently deletes the file, without going to a recycle bin

Reply:
Updated, let me know if you want additional clarification.

```
Delete a file (permanently).
```